src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -87,13 +81,11 @@ fun insertNewTraitMembers(selected: Collection<RsTraitMemberChooserMember>, impl
             selected.mapNotNull { it.member as? RsConstant }
     )
     val lastMethodOrBrace = impl.functionList.lastOrNull() ?: impl.lbrace ?: return
-    runWriteAction {
-        impl.addRangeAfter(
-                templateImpl.lbrace?.nextSibling,
-                templateImpl.rbrace?.prevSibling,
-                lastMethodOrBrace
-        )
-    }
+    impl.addRangeAfter(
```

Comment:
Should be fixed in: https://github.com/intellij-rust/intellij-rust/commit/c8811197553f69257fb53721dac52071fa128fe2